pkg/scheduler/framework/v1alpha1/waiting_pods_map.go

```diff
@@ -132,7 +132,7 @@ func (w *waitingPod) Allow(pluginName string) bool {
 
 	// Only signal success status after all plugins have allowed
 	if len(w.pendingPlugins) != 0 {
-		return true
+		return false
```

Comment:
I think the semantics of the return value is not quite clear. Perhaps we should not return anything. ditto the Reject function.

Reply:
Agree. Let me update.